pkg/scheduler/factory/factory.go

```diff
@@ -249,6 +249,11 @@ func NewConfigFactory(args *ConfigFactoryArgs) Configurator {
 		stopEverything = wait.NeverStop
 	}
 	schedulerCache := internalcache.New(30*time.Second, stopEverything)
+	// TODO(bsalamat): config files should be passed to the framework.
+	framework, err := framework.NewFramework(args.Registry, nil, schedulerCache)
```

Comment:
How do you plan to read config files here and would there be any validation for the config passed and would it be versioned?

Reply:
The KEP of the framework has some information about the config file. It will be similar to the scheduler/extender config and it will be versioned.